lizmap/modules/lizmap/lib/Commands/WMTSSeed.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Lizmap\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WMTSSeed extends \Jelix\Scripts\ModuleCommandAbstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('wmts:cache:seed')
+            ->setDescription('Generate cache for a layer')
+            ->setHelp('verbose option will provide more info')
+            ->addArgument('repository', InputArgument::REQUIRED, 'The repository ID')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project name')
+            ->addArgument('layers', InputArgument::REQUIRED, 'Coma separated layers list or \'*\' ')
```

Comment:
```suggestion
            ->addArgument('layers', InputArgument::REQUIRED, 'Comma separated layers list or \'*\' ')
```

L25 as well a few lines below
Not sure you want a coma ;-)

As you are in this area, can you harmonize capital versus lower case. Netpicking, but it would be nice to have a single rule ;-)